scene/gui/popup_menu.h

```diff
@@ -222,6 +223,7 @@ class PopupMenu : public Popup {
 	void set_item_text_direction(int p_idx, Control::TextDirection p_text_direction);
 	void set_item_language(int p_idx, const String &p_language);
 	void set_item_icon(int p_idx, const Ref<Texture2D> &p_icon);
+	void set_item_icon_max_width(int p_idx, int p_width);
```

Comment:
I think it would be great if this was exposed (like it's already in TreeItem).
I needed it at least once in my projects.